docs/DeveloperGuide.adoc

```diff
@@ -732,9 +732,67 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
 
 |`*` |user with many persons in the address book |sort persons by name |locate a person easily
-|=======================================================================
 
-{More to be added}
+|`* * *` |new user |see suggested commands as I type them |easily remember what to write without referring to the help sheet
+
+|`* * *` |user |edit information about a contact without retyping everything |save time and work
+
+|`* * *` |user |add or delete multiple users in one go |I can save time by not repeating commands
+
+|`* * *` |user |tag people using any tag |organize my contacts and add other pertinent information about them
+
+|`* * *` |user |use case-insensitive commands |type or retype commands in a specific case
+
+|`* * *` |user |add contacts while leaving one or more fields blank, except for the name |save contacts without having to know every detail
+
+|`* * *` |user |search my contacts by name, number, tag or address |find people more easily
+
+|`* * *` |user |conduct the search quickly and efficiently |save time
```

Comment:
`conduct the search quickly and efficiently` how? 
This is a case where your user story is not clearly specifying what the user wants

Reply:
@eryao95